tests/extract_tests.ml

```diff
@@ -8,7 +8,7 @@ let protect ~finally f =
       finally ();
       r
 
-let disabled = [ 215; 216; 519 ]
```

Comment:
💪💪

Reply:
To be fair though, I already gave a try at fixing the remaining one but it definitely doesn't feel easy 😬. I'll continue to investigate but I can't promise anything 😅